app/assets/stylesheets/navbar.scss

```diff
@@ -28,4 +28,22 @@
   .navbar-collapse {
     border-top: 0;
   }
+
+  @media screen and (min-width: 768px) and (max-width: 991px){
```

Comment:
It would be better if we could have just `min-width` here because having `max-width` values in addition to `min-width` can be a real pain to debug, especially if contributors aren't sure if we're using a mobile-first or desktop-first approach. It's fine to leave this as is for now, though. This will be fixed in the future.

Also, we should be using `em` values for media queries. However, since the site is currently using `px` values, let's stick with `px` for now until we make that change site-wide.